proto/cosmos/crypto/sr25519/keys.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package cosmos.crypto.sr25519;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/crypto/keys/sr25519";
+
```

Comment:
```suggestion
option (gogoproto.messagename_all) = true;
option (gogoproto.goproto_stringer_all) = false;
option (gogoproto.goproto_getters_all) = false;

```

Reply:
This will optimize few things. I think we will need to manally implement String method for `PrivKey` as well (we don't want to print private key - so just print `typename{-}`